frontend/spec/features/merge_and_transfer_spec.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails_helper'
+
+describe 'Merging and transfering resources', js: true do
+  let(:admin) { BackendClientMethods::ASpaceUser.new('admin', 'admin') }
+
+  before(:all) do
+    now = Time.now.to_i
+    @repository_source = create(:repo, repo_code: "transfer_test_source_#{now}")
+    @repository_target = create(:repo, repo_code: "transfer_test_target_#{now}")
+
+    set_repo @repository_source
+  end
+
+  before(:each) do
+    login_user(admin)
+  end
+
+  it 'can transfer a resource to another repository and open it for editing' do
+    select_repository(@repository_source)
+
+    resource = create(:resource)
+    run_index_round
+
+    visit "resources/#{resource.id}/edit"
+    find('#transfer-dropdown button').click
+    select @repository_target.repo_code, from: 'transfer_ref_'
+
+    within '.dropdown-menu.open-aligned-right.transfer-form' do
+      click_on 'Transfer'
+    end
+
+    within '#confirmChangesModal' do
+      click_on 'Transfer'
+    end
+
+    element = find('.alert.alert-success.with-hide-alert')
+    expect(element.text).to eq 'Transfer Successful. Records may take a moment to appear in the target repository while re-indexing takes place.'
+
+    run_all_indexers
+
+    visit '/'
+    select_repository(@repository_target)
+    element = find('.alert.alert-success.with-hide-alert')
+    expect(element.text).to eq "The Repository #{@repository_target.repo_code} is now active"
+
+    click_on 'Browse'
+    click_on 'Resources'
+
+    expect(page).to have_css 'tr', text: resource.title
+  end
+
+  it 'can merge a resource into a resource' do
```

Comment:
Minor nitpick but the word "merge" here should really be "transfer" since that is the language of the action taken and the confirmation text, and there is precedence for it elsewhere.